src/cosimulation.cc

```diff
@@ -23,8 +55,12 @@ ComplexCoDRAMsim3::~ComplexCoDRAMsim3() {
 }
 
 void ComplexCoDRAMsim3::tick() {
-    memory->ClockTick();
-    dram_clock++;
+    cpu_time += CPU_CLOCK_CYCLE;
+    if (cpu_time >= next_dram_cycle_time) {
```

Comment:
Should this be `while`? If CPU frequency is slower than the DRAM frequency, there may be multiple DRAM ticks at one CPU cycle.

Reply:
Yes